### ISSUE TITLE
specifying modes train, validation and test

### ISSUE THREAD

**Issue by fansariadeh** [tkipf/pygcn#60]
Hi guys!

I hope any of you happen to know why in line 88 of train.py, while it tries to specifies test mode, we see these line:
def test():
    model.eval()  
Dont you think it should be ###  model.test() ### sine we are focusing on test set?

Thank you in advance guys.

**Comment by chao1224** [comment]
@fansariadeh There's no `model.test()` in pytorch.
When you train the model, call `model.train()` will activate some layers like batch-norm, zero-out.
When you test the model, call `model.eval()` will modify the operations through these layers.
